app/services/arcgis_api/mock/responses/request_suggestions_response.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
a mock response like this should definitely live in `spec/support/fixtures` not under the `app` dir